src/engine/Filter.cpp

```diff
@@ -400,8 +416,9 @@ vector<RT>* Filter::computeFilterFixedValueForResultType(
               data->begin(), data->end(), rhs_array,
               [lhs](const RT& l, const RT& r) { return l[lhs] < r[lhs]; });
```

Comment:
`ValueReader`?

Reply:
The prefix filter is currently only run on KB type columns, which can be interpreted directly.

Reply:
I see